var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -241,6 +241,7 @@ def install(self, spec, prefix):
             "-Cbuilddir=build",
             "--no-isolation",
             "--skip-dependency-check",
+            "-Ccompile-args=-j%s" % make_jobs,
```

Comment:
Is `make_jobs` provided by spack?

Reply:
Yep (through `from spack.package import *`)